parse_results.py

```
@@ -1,6 +1,6 @@
 import json
 
-data = json.load(open("egoschema_subset.json"))
+data = json.load(open("DataSet/egoschema_subset.json"))
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)